resources/templates/comp/modal_vanity.twig

```diff
@@ -0,0 +1,19 @@
+<div class="modal fade" id="modalVanity" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-sm">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">{{ lang('vanity_url') }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <input type="text" class="form-control" id="modalVanity-input" >
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I would limit this to 64 chars also on the frontend, since is currently the hard limit

Reply:
Interesting, I did not know this was a hard limit with XBackbone. I thought it would be cool if users could make a vanity url as long as they like, but it makes since why this is disallowed. Fixed :+1: